features/src/mambaforge/install.sh

```diff
@@ -23,7 +23,7 @@ wget --no-hsts -q -O /tmp/miniforge.sh \
 
 echo "Installing Mambaforge...";
 
-# Install miniconda
+# Install Mambaforge
```

Comment:
We need to replace mambaforge with miniforge everywhere in this repo.

Reply:
Yes, but I don't want to take that on in this PR. Please raise a new issue/PR

Reply:
Okay. We can merge as-is for now.